NEXT_CHANGELOG.md

```diff
@@ -41,6 +41,14 @@ It defaults to 30 seconds.
 
 By [@o0Ignition0o](https://github.com/o0Ignition0o) in https://github.com/apollographql/router/pull/2271
 
+## 🐛 Fixes
+### Traces won't cause missing field-stats ([Issue #2277](https://github.com/apollographql/router/issues/2267))
```

Comment:
```suggestion
### Traces won't cause missing field-stats ([Issue #2267](https://github.com/apollographql/router/issues/2267))
```

Reply:
Fixed in [8159ad8](https://github.com/apollographql/router/pull/2277/commits/8159ad89e05bca7df75c7a1131c29a41e91cec7d)